src/components/signUpForm/SignUpForm.tsx

```diff
@@ -0,0 +1,188 @@
+import {useEffect, useState} from 'react';
+import {Image, ScrollView, View} from 'react-native';
+
+import {Button} from '@src/components/button';
+import {Input} from '@src/components/input';
+import {
+  alphaRegex,
+  emailRegex,
+  genderData,
+  passwordRegex,
+} from '@src/constants/constant';
+import {getData, storeData} from '@src/storage/storage';
+import RadioButton from '@src/components/radioButton';
```

Comment:
instead of `@src`, define alias such as `@components`, `@storage`, `@contexts`, etc.